Services/Administration/OJS.Services.Administration.Business/IParticipantsBusinessService.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OJS.Services.Administration.Business;
+
+using SoftUni.Services.Infrastructure;
+using System.Threading.Tasks;
+
+public interface IParticipantsBusinessService : IService
+{
+    /// <summary>
+    /// This method updates the total score snapshot for all participants.
+    /// </summary>
+    Task UpdateTotalScoreSnapshotOfParticipants();
+
+    /// <summary>
+    /// This method removes multiple scores associated with a participant
+    /// For each group, it keeps the score with the highest points and prepares to remove the rest.
+    /// </summary>
```

Comment:
No need to start with "This method". Just starting with "Removes... " is good enough, as it's clear that this is method.
Also does it just prepare to remove, or actually is meant to save the changes in the database? "Prepares" implies that it will not save changes.
Another point to be made - multiple scores for same participant is allowed, make sure it is mentioned that multiple **duplicated** scores are removed, meaning multiple scores for the same Problem and IsOfficial for that participant. Might be good idea to rename the method also to reflect this.